### PR TITLE
Enhancements to User Guide and Index

### LINKED ISSUE #139: [User Guide] Dropdown does not work in PDF format
image

### PR DESCRIPTION
**Pull request by itsme-zeix**
Includes some styling changes to the CSS files directly, mostly just reducing the excessive spacing between headers.

Do let me know if there are any other style changes you would like to see! Maybe we can discuss them and open an issue for it.

Closes #139 #141 #142 #143

### CODE REVIEW

**Review by ZShunRen — APPROVED** [review]

**Review by FionaQY — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -313,9 +323,9 @@ add n/ <NAME> p/ <PHONE> e/ <EMAIL> a/ <ADDRESS> j/ <JOBNAME> i/ <INCOME> [t/ <T
...
 
 **Command Format:**
-  ```
-  add n/ <NAME> p/ <PHONE> e/ <EMAIL> a/ <ADDRESS> j/ <JOBNAME> i/ <INCOME> [t/ <TIER>] [rn/ <REMARK>]
-  ```
+```
+add n/ <NAME> p/ <PHONE> e/ <EMAIL> a/ <ADDRESS> j/ <JOBNAME> i/ <INCOME> [t/ <TIER>] [rn/ <REMARK>]
+```
 * Mandatory Fields: `n/`, `p/`, `e/`, `a/`, `j/`, `i/`
 * Optional Fields: `t/`, `rn/`
```
Comment (colinhia) [inline docs/UserGuide.md:330]: In line with the Bug found earlier, rn/ should be replaced with r/

File: docs/UserGuide.md [hunk 2]
```diff
@@ -718,6 +730,7 @@ Each credit card tier is visually distinguished in the UI: Gold is marked with a
 
 --------------------------------------------------------------------------------------------------------------------
 
+
 ## 8. Command Summary
 
 | **Action**                 | **Command Format**                                                                                                           | **Example**                                                                                               |
```
Comment (colinhia) [inline docs/UserGuide.md:736]: Similarly, the flag in the add command summary should be reflected as r/

**Review by colinhia — APPROVED** [review]
LGTM! Just some minor catches with the r/ flag for add

**Pull request merged**